web-ng/src/app/app.module.ts

```diff
@@ -35,6 +35,11 @@ import { ShareDialogModule } from './components/share-dialog/share-dialog.module
 import { HttpClientModule } from '@angular/common/http';
 import { firebase, firebaseui, FirebaseUIModule } from 'firebaseui-angular';
 import { TitleDialogModule } from './components/title-dialog/title-dialog.module';
+import { MatCardModule } from '@angular/material/card';
+import {MatGridListModule} from '@angular/material/grid-list';
```

Comment:
Please enable Prettier "format on save" in Visual Studio (or equivalent) to ensure consistent formatting.